README.md

```diff
@@ -156,8 +156,8 @@ curl --location --request GET 'http://localhost:8038/api/v1/vouchers?guid=<guid>
 Post the Voucher to RV and Owner Server
 Post the fetched voucher to the RV and Owner server using curl:
 ```
-curl -X POST 'http://localhost:8041/api/v1/owner/vouchers' -d @ownervoucher
-curl -X POST 'http://localhost:8043/api/v1/owner/vouchers' -d @ownervoucher
+curl -X POST 'http://localhost:8041/api/v1/owner/vouchers' --data-binary @ownervoucher
```

Comment:
Not a review comment but a question for my own benefit: why load the ownervoucher to the rendezvous server? IIUC the owner passes the voucher to the rv server on TO0 so why is it necessary to load the voucher explicitly?

Reply:
since this is just changing something already there, I'll open a simple issue to get rid of this (as this predates this code change)